recurly.module

```diff
@@ -139,6 +139,11 @@ function recurly_entity_update(\Drupal\Core\Entity\EntityInterface $entity) {
 
   // Check if any of the mapping tokens have changed.
   $original_entity = \Drupal::entityManager()->getStorage($entity_type)->load($entity->getOriginalId());
+  // Only makes sense to continue if $original_entity is not null.
+  if (is_null($original_entity)) {
```

Comment:
@markdorison I'm not sure why, but that single line implementation doesn't work. I wrap all the remaining code inside of your suggested `if`, but the same ole' fatal error is thrown.